File: tests/Kros.KORM.UnitTests/KormTestsCollection.cs
using Xunit;

namespace Kros.KORM.UnitTests;

[CollectionDefinition(Name)]
public class KormTestsCollection : ICollectionFixture<KormTestsFixture>
{
    // This class has no code, and is never created. Its purpose is simply
    // to be the place to apply [CollectionDefinition] and all the
    // ICollectionFixture<> interfaces.
    public const string Name = "KormUnitTests";
}

File: tests/Kros.KORM.VB.UnitTests/KormTestsCollection.vb
Imports Kros.KORM.UnitTests
Imports Xunit

Namespace Kros.KORM.VB.UnitTests

    <CollectionDefinition(KormTestsCollection.Name)>
    Public Class KormTestsCollection
        Implements ICollectionFixture(Of KormTestsFixture)

        ' This class has no code, And Is never created. Its purpose Is simply
        ' to be the place to apply [CollectionDefinition] And all the
        ' ICollectionFixture<> interfaces.
        Public Const Name As String = "KormUnitTests"
    End Class

End Namespace
